projects/extension/src/components/Bootnodes.tsx

```diff
@@ -0,0 +1,262 @@
+import React, { useEffect, useState } from "react"
+import { MdDeleteOutline } from "react-icons/md"
+import * as environment from "../environment"
+
+import "./Bootnodes.css"
+import { Title, Switch } from "."
+interface BootnodesType {
+  checked: boolean
+  bootnode: string
+}
+
+interface Boot {
+  [key: string]: string[]
+}
+
+// Add to localstorage the given bootnode for the given chain
+const saveToLocalStorage = async (
+  chainName: string,
+  bootnode: string,
+  add: boolean,
+  def: string[],
+) => {
+  if (def.length === 0) throw new Error("Default Bootnodes should exist.")
+  let res: string[]
+  const chainBootnodes = await environment.get({
+    type: "bootnodes",
+    chainName,
+  })
+  res =
+    chainBootnodes && Object.keys(chainBootnodes).length > 0
+      ? [...chainBootnodes]
+      : [...def]
+  add ? res.push(bootnode) : res.splice(res.indexOf(bootnode), 1)
+  environment.set({ type: "bootnodes", chainName }, res)
+}
+
+export const Bootnodes = () => {
+  const [selectedChain, setSelectedChain] = useState<string>("polkadot")
+  const [defaultBn, setDefaultBn] = useState<BootnodesType[]>([])
+  const [customBn, setCustomBn] = useState<BootnodesType[]>([])
+  const [customBnInput, setCustomBnInput] = useState<string>("")
+  const [defaultWellKnownChainBn, setDefaultWellKnownChainBn] = useState<
```

Comment:
```suggestion
  const [selectedChainDefaultBn, setSelectedChainDefaultBn] = useState<
```

You use the terminology "selectedChain", so let's be consistent